Petty/Extensions/UIView+.swift

```diff
@@ -21,7 +25,10 @@ extension UIView {
         layer.rasterizationScale = scale ? UIScreen.main.scale : 1
     }
 
-    func roundCorner(color: UIColor, cornerRadius: CGFloat, shadowRadius: CGFloat, clipsToBounds: Bool) {
+    func roundCorner(color: UIColor,
+                     cornerRadius: CGFloat,
+                     shadowRadius: CGFloat,
+                     clipsToBounds: Bool) {
         self.clipsToBounds = true
```

Comment:
```suggestion
        clipsToBounds = true
```

Reply:
Cannot assign to value: 'clipsToBounds' is a 'let' constant
Add explicit 'self.' to refer to mutable property of 'UIView'

Xcode báo phải để self